content/installation/java/configuration/YAML-properties-java.md

```diff
@@ -0,0 +1,204 @@
+<!--
+title: "YAML configuration properties for the Java agent "
+description: "Instructions for configuring Java agent properties via YAML file"
+tags: "installation java agent YAML configuration rules properties"
+-->
+
+Contrast supports YAML-based configuration for the Java agent. This allows you to store configuration on disk and override that configuration with environment variable and CLI args, if necessary.
+
+> **Note:** While property formatting in YAML configuration files is shared by all Contrast agents, each agent must use its specified file. 
+
+## Load Path
+
+Configuration values use the following order of precedence: 
+
+1. Corporate rule (e.g., expired license overrides `assess.enable`)
+1. System property value
+1. Environment variable value
+1. User configuration file value 
+<!-- TODO , for the java agent this includes the legacy file and the YAMl file. Should we specify both or just not mention the legacy file (which no one should be using anymore, I think.-->
+1. Contrast UI value
+1. Default value
+
+<!-- Other use and file location info. This will be done after CONTRAST-26125  -->
+
+Go to the [Java YAML Template](installation-netconfig.html#net-template) for fully formatted properties that you can copy and use in your own agent configuration files. 
+
+## Set as System Properties
+
+You can also set all of the following YAML properties as system properties. Derive the system property key from the YAML by joining every node with a "." until you reach the bottom property. 
+
+> **Example:** If you want to override the `contrast` property, as given below, you can set `-Dcontrast.enable=false` as a system property.
+  * contrast: 
+    * enable: `true`
```

Comment:
@palmergs Is the info in this "System Properties" section true for Ruby and Python as well?

Reply:
I don't think it is. These system properties are a java thing, and I'm not sure what the ruby equivalent is.